content/chainguard/chainguard-images/working-with-images/fips-images.md

```diff
@@ -18,29 +18,45 @@ weight: 050
 toc: true
 ---
 
-One of the primary requirements of federal compliance frameworks — including [FedRAMP](https://www.fedramp.gov/program-basics/) — is to use FIPS-validated cryptography. Chainguard offers FIPS-enabled versions of a number of its Images. This article provides a high-level overview of what FIPS is and what to expect from Chainguard’s FIPS-enabled Images.
+## What is FIPS? 
+
+One of the primary requirements of federal compliance frameworks — including [FedRAMP](https://www.fedramp.gov/program-basics/) — is to use FIPS-validated cryptography. To help customers meet these requirements, Chainguard offers FIPS-enabled versions of many images. This article provides a high-level overview of what FIPS is, what to expect from Chainguard FIPS Images, featuring a kernel-independent design, and how Chainguard FIPS images stand out from alternatives.
 
 [Federal Information Processing Standards](https://www.nist.gov/itl/publications-0/federal-information-processing-standards-fips) (FIPS) are publicly announced standards developed by the National Institute of Standards and Technology (NIST) in accordance with the Federal Information Security Management Act (FISMA) and approved by the Secretary of Commerce. FIPS compliance ensures that cryptographic security services within applications meet strict security and integrity standards, and are implemented and configured correctly.
 
+## What To Expect from Chainguard FIPS Images
+
 ‍Chainguard warranties the following with respect to Chainguard container images:
 
-Chainguard’s FIPS Images available to be delivered in compliance with FIPS specifications are listed [here](https://images.chainguard.dev/?category=fips)  (each a "Chainguard FIPS Image"). Images will be made available in compliance with FIPS specifications provided a customer’s applicable order form designates the purchase of Chainguard FIPS images.
+Chainguard FIPS Images available to be delivered in compliance with FIPS specifications are listed [here](https://images.chainguard.dev/?category=fips)  (each a "Chainguard FIPS Image"). Images will be made available in compliance with FIPS specifications provided a customer’s applicable order form designates the purchase of Chainguard FIPS images.
 
 The Chainguard FIPS images contain FIPS-validated software cryptographic modules. Entropy must be provided as specified in its cryptographic policy. The cryptographic module may provide non-approved algorithms, which will result in operating in FIPS non-approved mode. The cryptographic FIPS modules currently provided are:
 
-‍- OpenSSL FIPS Provider (CMVP [#4282](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282))
+- OpenSSL FIPS 3.0 Provider Module (CVMP [#4856] (https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4856))
```

Comment:
BTW there's a rogue space after the ] which breaks the link

Reply:
Thank you!

Reply:
Another thing; I think it should be "Chainguard OpenSSL FIPS 3.0 Provider Module", just to make it clear that it's our own cert (unlike the bouncy castle ones). This also matches the text in the link.